ios/FluentUI.Demo/FluentUI.Demo/DemoController.swift

```diff
@@ -38,12 +38,9 @@ class DemoController: UIViewController {
 
     var allowsContentToScroll: Bool { return true }
 
-    func createButton(title: String, action: Selector) -> MSFButton {
-        let button = MSFButton()
-        button.titleLabel?.textAlignment = .center
-        button.titleLabel?.numberOfLines = 0
-        button.setTitle(title, for: .normal)
-        button.addTarget(self, action: action, for: .touchUpInside)
+    func createButton(title: String, action: (( MSFButtonVnext) -> Void)?) -> MSFButtonVnext {
```

Comment:
Do you think it would be better to use an extension? This base class `DemoController` fixes problem in one place and creates in another

Reply:
To be honest I was about to just delete this func and create the buttons directly since it only provides the convenience of the default style/size and setting the text.

Usually we prefer standalone funcs (not instance funcs like this) instead of extensions.
But the main reason I opted to just replace the implementation of this one is that we have other convenience "create" funcs here.